src/raftstore/store/store.rs

```diff
@@ -534,6 +544,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         self.apply_res_receiver = Some(rx);
         box_try!(self.apply_worker.start(apply_runner));
 
+        let (tx, rx) = mpsc::channel();
+        let space_check_runner = SpaceCheckRunner::new(Arc::clone(&self.kv_engine), tx);
```

Comment:
Can we merge this worker with the `compact_worker`? It seems overwhelmed to create a worker for each task.
BTW, can you use the `sendch` in the raft store to send back the result?

Reply:
done